tests/test_ingestion.py

```diff
@@ -471,6 +472,19 @@ def testFindMatchingFiles(self):
             allFiles
         )
 
+    def testPickling(self):
+        """Test that a Gen3DatasetIngestTask can be pickled correctly.
+
+        This is needed for multiprocessing support.
+        """
+        stream = pickle.dumps(self.task)
+        copy = pickle.loads(stream)
+        self.assertEqual(self.task.getFullName(), copy.getFullName())
+        self.assertEqual(self.task.log.getName(), copy.log.getName())
+        # Equality for config ill-behaved; skip it
```

Comment:
This is a minor nitpick, but could you rephrase this comment? "Skip it" suggests to me that code including a test for equality of the config is present and is skipped, while what is happening here is that code doesn't exist (for a good reason). Maybe something like "Do not test equality for the config because it is ill-behaved"?

Reply:
IIRC `Config.__eq__` does exist, but it doesn't handle `ConfigurableField` very well. I probably should have filed a ticket, but was too distracted at the time.